docs/source/tools/sandbox.rst

```diff
@@ -48,6 +48,10 @@ To start Sandbox using persistence, pass an ``--sql-backend-jdbcurl <value>`` op
 
 Here is an example for such a url: ``jdbc:postgresql://localhost/test?user=fred&password=secret``
 
+Due to possible conflicts between the ``&`` character and various terminal shells, we recommend quoting the jdbc url like so:
+
+  $ daml sandbox Main.dar --sql-backend-jdbcurl "jdbc:postgresql://localhost/test?user=fred&password=secret"
```

Comment:
Having already approved, I'd strongly recommend using the `code-block` directive over the colon + indentation syntax, as Sphinx tends to use default (Python) highlighting for the latter.